src/components/ItemList.js

```diff
@@ -1,5 +1,44 @@
+import { collection, query, where, onSnapshot } from 'firebase/firestore';
+import { useEffect, useState } from 'react';
+import { db } from '../lib/firebase.js';
+import { Link } from 'react-router-dom';
+
 const ItemList = () => {
-  return <h1>Item List</h1>;
+  const token = 1234;
```

Comment:
Same here, I believe. I also replaced line 7 on my machine with:
const token = localStorage.getItem('token');

I then navigated to the shopping list and only am seeing the item that matches my local storage token!
<img width="311" alt="Screen Shot 2022-01-19 at 12 45 19 PM" src="https://user-images.githubusercontent.com/61766455/150211169-cca563c6-98d9-42a5-811d-d44edd36b432.png">